tools/run-fs-offline.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+# Copyright (c) 2022 The MobileCoin Foundation
+
+NET="$1"
+
+if [ "$NET" == "main" ]; then
+    NAMESPACE="prod"
+    PEER_DOMAIN="prod.mobilecoinww.com/"
+    TX_SOURCE_URL="https://ledger.mobilecoinww.com"
+    INGEST_SIGSTRUCT_URI=$(curl -s https://enclave-distribution.${NAMESPACE}.mobilecoin.com/production.json | grep ingest-enclave.css | awk '{print $2}' | tr -d \" | tr -d ,)
+elif [ "$NET" == "test" ]; then
+    NAMESPACE=$NET
+    PEER_DOMAIN="test.mobilecoin.com/"
+    TX_SOURCE_URL="https://s3-us-west-1.amazonaws.com/mobilecoin.chain"
+    INGEST_SIGSTRUCT_URI=$(curl -s https://enclave-distribution.${NAMESPACE}.mobilecoin.com/production.json | grep ingest-enclave.css | awk '{print $2}' | tr -d \" | tr -d ,)
```

Comment:
I think it would be nice to future devs to remove the unnecessary code from this script